connect-inject/endpoints_controller.go

```diff
@@ -166,13 +173,9 @@ func (r *EndpointsController) Reconcile(ctx context.Context, req ctrl.Request) (
 
 					// Update the TTL health check for the service.
 					// This is required because ServiceRegister() does not update the TTL if the service already exists.
-					status, reason, err := getReadyStatusAndReason(pod)
-					if err != nil {
-						r.Log.Error(err, "failed to get status and reason from pod", "name", serviceRegistration.Name)
-						return ctrl.Result{}, err
-					}
-					r.Log.Info("updating TTL health check for service", "name", serviceRegistration.Name, "reason", reason, "status", status)
-					err = client.Agent().UpdateTTL(getConsulHealthCheckID(pod, serviceRegistration.ID), reason, status)
+					reason := getHealthCheckStatusReason(address, healthStatus)
+					r.Log.Info("updating TTL health check for service", "name", serviceRegistration.Name, "reason", reason, "status", healthStatus)
```

Comment:
nit: I don't think users care (or know) what a TTL health check is. I think it would be fine to say "updating health check status"

Reply:
yeah let me change that.